src/main/java/com/w6/controller/EndpointController.java

```diff
@@ -25,6 +26,7 @@
     protected static final String QUERY_VIEW = "query";
     protected static final String DOCUMENTS_BY_EVENT_VIEW = "articlesOfEvent";
     protected static final String REPORT_VIEW = "report";
+    protected static final String EMAILS_VIEW = "emails";
```

Comment:
It should be private, if you are not garantee that this field is working fine with inheretance, you should use private.